src/machine/board_arduino_zero.go

```diff
@@ -0,0 +1,100 @@
+// +build sam,atsamd21,arduino_zero
+
+package machine
+
+// used to reset into bootloader
+const RESET_MAGIC_VALUE = 0x07738135
+
+// GPIO Pins - Digital Low
+const (
+	D0 = PA11 // RX
+	D1 = PA10 // TX
+	D2 = PA14
+	D3 = PA09 // PWM available
+	D4 = PA08 // PWM available
+	D5 = PA15 // PWM available
+	D6 = PA20 // PWM available
+	D7 = PA21
+)
+
+// GPIO Pins - Digital High
+const (
+	D8  = PA06 // PWM available
+	D9  = PA07 // PWM available
+	D10 = PA18 // PWM available
+	D11 = PA16 // PWM available
+	D12 = PA19 // PWM available
+	D13 = PA17 // PWM available
+)
+
+// LEDs on the Arduino Zero
+const (
+	LED  Pin = D13
+	LED1 Pin = PA27 // TX LED
+	LED2 Pin = PB03 // RX LED
+)
```

Comment:
So far we've made `LED` an alias of `LED1`, I think it would be a good idea to keep that.

```suggestion
const (
	LED      = LED1
	LED1 Pin = D13
	LED2 Pin = PA27 // TX LED
	LED3 Pin = PB03 // RX LED
)
```